rules/ak/ak-qx.js

```diff
@@ -0,0 +1,24 @@
+( function ( $ ) {
+	'use strict';
+
+	var defs = {
+		id: 'ak-qx',
+		name: 'Akan - QX replacement',
+		description: 'Akan input method with Q and X replaced by Ɛ and Ɔ',
+		date: '2016-06-23',
+		URL: 'http://www.kasahorow.org/node/260',
+		author: 'Amir E. Aharoni, based on Kasahorow',
+		license: 'GPLv3',
```

Comment:
Is it according to any standard?
All other files use "GPLv3". Should it be replace everywhere? If yes, it should be done in another patch.